CV - Whats supported (#2467)
harness/developer-hub: docs/continuous-delivery/verify/cv-whats-supported.md

sidebar_position: 10
---

Harness Continuous Verification (CV) supports the following health and change monitoring tools:

## Health sources

A health source monitors health trends in a service by analyzing metrics from APM tools and logs from log management tools.

### APM

Harness CV supports the following APM tools:

- AppDynamics
- Amazon CloudWatch
- Custom health source
- Datadog
- Dynatrace
- Google Cloud Operations (formerly Stackdriver)
- New Relic
- Prometheus (includes support for AWS Prometheus)
- SignalFx
- Splunk Observability
- Sumologic

> Harness supports most of the popular APM tools, but there may be an instance where Harness does not have a native connector. Using the **Harness Custom Health Source** feature, you can integrate such APM tools with Harness.

Reviewer: ```suggestion
Harness supports most of the popular APM tools, but there may be instances where Harness does not have a native connector. Using the **Harness Custom Health Source** feature, you can integrate such APM tools with Harness.
```

### Log management

Harness CV supports the following log management tools:

- Datadog
- Elasticsearch
- Google Cloud Operations (formerly Stackdriver)
- Grafana Loki
- Splunk Observability
- Sumologic

> Harness supports most of the popular log management tools, but there may be an instance where Harness does not have a native connector. Using the **Harness Custom Health Source** feature, you can integrate such log management tools with Harness.

Reviewer: ```suggestion
Harness supports most of the popular log management tools, but there may be instances where Harness don't have a native connector. Using the **Harness Custom Health Source** feature, you can integrate such log management tools with Harness.
```
